packages/analyticsdx-template-lint/test/unit/validator.test.ts

```diff
@@ -193,22 +193,29 @@ describe('FileTemplateValidator', () => {
     expect(diagnostics, 'diagnostics').to.deep.equal({
       [templateInfoPath]: [
         {
-          range: { start: { line: 13, character: 2 }, end: { line: 13, character: 42 } },
+          range: { start: { line: 14, character: 2 }, end: { line: 14, character: 42 } },
```

Comment:
When I added `layoutDefinition` to the test template-info.json file, a bunch of the other existing diagnostics got moved a line down.